src/Routes/RosSystemDetail/RosSystemDetail.js

```diff
@@ -134,7 +135,7 @@ class RosSystemDetail extends React.Component {
                                 <Main>
                                     <Grid gutter="md">
                                         <GridItem span={12}>
-                                            <AppInfo showTags fallback="" />
+                                            { entity ? <SystemDetailWrapper showTags fallback=""/> : <Spinner/>}
```

Comment:
In case SystemDetail is independent of the header, can we render SystemDetailWrapper directly without waiting for entity?

Reply:
Aha, I understand. Thanks for letting me know